android/quest/src/main/java/org/smartregister/fhircore/quest/ui/profile/ProfileScreen.kt

```diff
@@ -154,7 +152,10 @@ fun ProfileScreen(
           color = MaterialTheme.colors.primary,
         )
       }
-      LazyColumn(state = lazyListState) {
+      LazyColumn(
+        state = lazyListState,
+        modifier = Modifier.padding(bottom = if (!fabActions.isNullOrEmpty() && fabActions.first().visible) 80.dp else 32.dp)
```

Comment:
Convert these to constants with intuitive names e.g `FOOTER_PADDING_WITH_FAB`, `FOOTER_PADDING_WITHOUT_FAB`